internal/service/api/handlers/register.go

```diff
@@ -105,7 +105,7 @@ func Register(w http.ResponseWriter, r *http.Request) {
 		types.NewCancunSigner(NetworkConfig(r).ChainID),
 		&types.LegacyTx{
 			Nonce:    NetworkConfig(r).Nonce(),
-			Gas:      gas,
+			Gas:      uint64(float64(gas) * NetworkConfig(r).GasMultiplier),
```

Comment:
check also below

Reply:
moved gasPrice multiplier for better clarifying